src/sdkGenerator.spec.ts

```diff
@@ -1,25 +1,32 @@
 import * as Models from './sdkModels'
-import { SdkGenerator } from './sdkGenerator'
+import { MethodGenerator } from './sdkGenerator'
 import { PythonGen } from './python.gen'
 
 const apiModel = Models.ApiModel.fromFile('./Looker.3.1.oas.json', './Looker.3.1.json')
 
 describe('sdk generator test', () => {
   it('outputs a method in Python', () => {
-    const gen = new SdkGenerator(apiModel, new PythonGen(apiModel))
-    const result = gen.codeFormatter.declareMethod('  ', apiModel.methods['create_look'])
+    const gen = new MethodGenerator(apiModel, new PythonGen(apiModel))
+    const result = gen.codeFormatter.declareMethod('', apiModel.methods['create_look'])
     expect(result).toEqual(
-      `  # POST /looks -> models.LookWithQuery
-  def create_look(
-      self,
-      body: Optional[models.WriteLookWithQuery] = None,
-      # Requested fields.
-      fields: Optional[str] = None
-  ) -> models.LookWithQuery:
-      """Create Look"""
-      response = self.post(f"/looks", models.LookWithQuery, query_params={"fields": fields}, body=body)
-      assert isinstance(response, models.LookWithQuery)
-      return response`)
+      `# POST /looks -> models.LookWithQuery
+def create_look(
+    self,
+    body: Optional[models.WriteLookWithQuery] = None,
+    # Requested fields.
+    fields: Optional[str] = None,
+    transport_options: Optional[transport.TransportSettings] = None,
+) -> models.LookWithQuery:
+    """Create Look"""
+    response = self.post(
+                        f"/looks",
+            models.LookWithQuery,
+            query_params={"fields": fields},
+            body=body,
+            transport_options=transport_options
+    )
+    assert isinstance(response, models.LookWithQuery)
+    return response`)
```

Comment:
yikes, sorry we broke this

Reply:
No big deal. We still have a moving target!